### DISCUSSION TITLE
How to delete playerdata or reset all claimblocks to zero

### DISCUSSION THREAD

**Discussion by eliyash31** [GriefPrevention/GriefPrevention#1053] [Q&A] [answered]
What command should I use?

**Comment by Jikoo** [comment, marked as answer]
That's not something you can do from in game.

Shut down your server, then
Flatfile:
Delete ClaimData and PlayerData out of GriefPreventionData
SQL:
Connect to your SQL database
run `DROP TABLE griefprevention_claimdata,griefprevention_playerdata,griefprevention_nextclaimid;`

If you don't want to delete admin claims in the process, you can use `/DeleteUserClaimsInWorld` from console for each world GP is enabled in prior to shutting down, then omit deleting claim data from the steps above.
